geomesa-convert/geomesa-convert-common/pom.xml

```diff
@@ -36,6 +40,14 @@
             <artifactId>geomesa-utils</artifactId>
         </dependency>
         <dependency>
+            <groupId>javax.media</groupId>
+            <artifactId>jai_core</artifactId>
```

Comment:
I removed imageio, as it wasn't needed, and verified that the cqs aren't affected by jai_core